library/Jira/MonitoringInfo.php

```diff
@@ -213,6 +223,8 @@ public function getDescriptionHeader()
 
         $description .= sprintf("Host: %s\n", $hostLink);
 
+        $description .= sprintf("Last icinga state change: %s\n", $this->getLastStateChange());
```

Comment:
Please remove "icinga". "Last state change:" is sufficient.

Reply:
Wouldn't this also have to be translated?

Reply:
Of course only if this is somehow displayed in Web. If this information is only transferred to Jira, this makes no sense :).